reference/commands/graph/info.rst

```diff
@@ -494,15 +372,12 @@ You can generate a graph of your dependencies, in ``dot`` or ``html`` formats:
     }
 
 
-
 .. note::
-    The generated *index.html* output file contains links to a third-party resource, the *vis.js* library (2 files: *vis.min.js*, *vis.min.css*).
-    By default they are retrieved from Cloudfare. However, for environments without internet connection, these files
-    could be also used from the local cache and installed with :command:`conan config install` by putting those
-    files in the root of the configuration folder:
-
-    - *vis.min.js*: Default link to "https://cdnjs.cloudflare.com/ajax/libs/vis/4.18.1/vis.min.js"
-    - *vis.min.css*: Default link to "https://cdnjs.cloudflare.com/ajax/libs/vis/4.18.1/vis.min.css"
-
-    It is not necessary to modify the generated html file. Conan will automatically use the local paths to the cache files if
-    present, or the internet ones if not.
+    If using ``format=html``, the generated html contains links to a third-party resource,
+    the *vis.js* library with 2 files: *vis.min.js*, *vis.min.css*.
+    By default they are retrieved from Cloudfare. However, for environments without internet connection,
+    you'll need to modify the template found in ``cli/formatters/graph/info_graph.html``
```

Comment:
You can't modify that, it is a Conan internal thing.
The template should be somewhere else inside ``<home>``